integreat_cms/cms/utils/round_up_hix_score.py

```diff
@@ -0,0 +1,8 @@
+def round_up_hix_score(raw_score: float | None) -> float | None:
+    """
+    Function to round up HIX score
+    """
+
+    if raw_score is None:
+        return None
+    return round(raw_score, ndigits=2)
```

Comment:
I would argue that this function could be better served by inlining it, as it is not particularly complex. This approach avoids the need for a new file (which readers would have to navigate to), and it results in fewer lines compared to its current implementation.

Reply:
I know your feeling 😅 Though I decided for a separate file because the function is used both in the model and the view and to avoid circular import...